pkg/controller/certificates/internal/test/test.go

```diff
@@ -72,14 +72,14 @@ type CryptoBundle struct {
 }
 
 func MustCreateCryptoBundle(t *testing.T, crt *cmapi.Certificate, fixedClock *fakeclock.FakeClock) CryptoBundle {
-	c, err := createCryptoBundle(crt, fixedClock)
+	c, err := CreateCryptoBundle(crt, fixedClock)
```

Comment:
Think I said this before, think this can be kept private

Reply:
Oops, this change isn't even relevant in this PR... my bad 😅